lib/index.js

```diff
@@ -11,27 +11,48 @@ function ThisData (apiKey, options) {
   assert(apiKey, 'You must pass your ThisData api key');
 
   var options = options || {};
-  this.apiKey = apiKey;
-  this.host = options.host || 'https://api.thisdata.com';
   this.verbs = verbs;
   this.event = new Event(options);
+
+  this.client = request.defaults({
+    headers: {
+      'User-Agent': 'thisdata-node'
+    },
+    baseUrl: options.host || 'https://api.thisdata.com/v1/',
```

Comment:
There might be a mismatch between what people expect `host` to be here and what is actually is, which is a base URL. We could consider renaming the `host` option to `endpoint` or `baseUrl` to make that clearer?

I think the way `this.send` worked was better in that regard